python-sdk/example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -43,8 +43,8 @@ def aggregate_data(df: pd.DataFrame):
     schedule_interval=None,
     default_args={
         "email_on_failure": False,
-        "retries": 0,
-        "retry_delay": timedelta(minutes=5),
+        "retries": 1,
```

Comment:
Why did we change 12 DAGs to have `"retries": 1`?

Reply:
@tatiana Earlier these DAG did not have retries before added to give one more retry in case of failure